wasm-client/deploy.js

```diff
@@ -55,23 +55,27 @@ async function deploy() {
     let accounts = await web3.eth.getAccounts()
     let fileSystem = await deployContract('Filesystem', {from: accounts[0], gas: 3500000})
     let judge = await deployContract('Judge', {from: accounts[0], gas: 4600000})
-    let merkle = await deployContract('Merkle', {from: accounts[0], gas: 1000000})
+    // let merkle = await deployContract('Merkle', {from: accounts[0], gas: 1000000})
     
     let interactive = await deployContract('Interactive', {from: accounts[0], gas: 3500000}, [judge._address])
-    await interactive.methods.registerJudge(1, merkle._address).send({from: accounts[0]})
+    // await interactive.methods.registerJudge(1, merkle._address).send({from: accounts[0]})
 
     let tru = await deployContract('TRU', {from: accounts[0], gas: 1000000})
     let exchangeRateOracle = await deployContract('ExchangeRateOracle', {from: accounts[0], gas: 1000000})
-    let incentiveLayer = await deployContract('IncentiveLayer', {from: accounts[0], gas: 4200000}, [tru._address, exchangeRateOracle._address, interactive._address, fileSystem._address])
+    let incentiveLayer = await deployContract('IncentiveLayer', {from: accounts[0], gas: 5200000}, [tru._address, exchangeRateOracle._address, interactive._address, fileSystem._address])
+    
+    accounts.forEach(async addr => {
```

Comment:
I'm skeptical this works as expected, as in it wont be synchronous, because forEach doesn't work that way. Might not need conventional for loop here

Reply:
Perhaps it's confusing (at least it confused me), but we just need to mint the test tokens for accounts, any order works.

Reply:
Right, didn't realize it was in deploy.js